d2core/d2input/ebiteninput.go

```diff
@@ -0,0 +1,226 @@
+package d2input
+
+import (
+	"github.com/hajimehoshi/ebiten"
+	"github.com/hajimehoshi/ebiten/inpututil"
+)
+
+func ebitenInput(im *inputManager) error {
+	im.key[Key0]				= int(ebiten.Key0)
+	im.key[Key1]				= int(ebiten.Key1)
+	im.key[Key2]				= int(ebiten.Key2)
+	im.key[Key3]				= int(ebiten.Key3)
+	im.key[Key4]				= int(ebiten.Key4)
+	im.key[Key5]				= int(ebiten.Key5)
+	im.key[Key6]				= int(ebiten.Key6)
+	im.key[Key7]				= int(ebiten.Key7)
+	im.key[Key8]				= int(ebiten.Key8)
+	im.key[Key9]				= int(ebiten.Key9)
+	im.key[KeyA]				= int(ebiten.KeyA)
+	im.key[KeyB]				= int(ebiten.KeyB)
+	im.key[KeyC]				= int(ebiten.KeyC)
+	im.key[KeyD]				= int(ebiten.KeyD)
+	im.key[KeyE]				= int(ebiten.KeyE)
+	im.key[KeyF]				= int(ebiten.KeyF)
+	im.key[KeyG]				= int(ebiten.KeyG)
+	im.key[KeyH]				= int(ebiten.KeyH)
+	im.key[KeyI]				= int(ebiten.KeyI)
+	im.key[KeyJ]				= int(ebiten.KeyJ)
+	im.key[KeyK]				= int(ebiten.KeyK)
+	im.key[KeyL]				= int(ebiten.KeyL)
+	im.key[KeyM]				= int(ebiten.KeyM)
+	im.key[KeyN]				= int(ebiten.KeyN)
+	im.key[KeyO]				= int(ebiten.KeyO)
+	im.key[KeyP]				= int(ebiten.KeyP)
+	im.key[KeyQ]				= int(ebiten.KeyQ)
+	im.key[KeyR]				= int(ebiten.KeyR)
+	im.key[KeyS]				= int(ebiten.KeyS)
+	im.key[KeyT]				= int(ebiten.KeyT)
+	im.key[KeyU]				= int(ebiten.KeyU)
+	im.key[KeyV]				= int(ebiten.KeyV)
+	im.key[KeyW]				= int(ebiten.KeyW)
+	im.key[KeyX]				= int(ebiten.KeyX)
+	im.key[KeyY]				= int(ebiten.KeyY)
+	im.key[KeyZ]				= int(ebiten.KeyZ)
+	im.key[KeyApostrophe]		= int(ebiten.KeyApostrophe)
```

Comment:
Weird indentation?

Reply:
Make sure your editor is honoring the .editorconfig file.

Reply:
it is, those were just logically distinct (keys vs mouse buttons), so i didn't feel like it needed the same indentation. I'll change it, though.